src/OrchardCore.Modules/OrchardCore.ReverseProxy/Drivers/ReverseProxySettingsDisplayDriver.cs

```diff
@@ -15,63 +16,56 @@ namespace OrchardCore.ReverseProxy.Drivers
 {
     public class ReverseProxySettingsDisplayDriver : SectionDisplayDriver<ISite, ReverseProxySettings>
     {
-        private const string RestartPendingCacheKey = "ReverseProxySiteSettings_RestartPending";
         private const string SettingsGroupId = "ReverseProxy";
-
-        private readonly INotifier _notifier;
-        private readonly IMemoryCache _memoryCache;
+        private readonly IShellHost _orchardHost;
+        private readonly ShellSettings _currentShellSettings;
 
         public ReverseProxySettingsDisplayDriver(INotifier notifier,
-            IMemoryCache memoryCache,
+            IShellHost orchardHost,
+            ShellSettings currentShellSettings,
             IHtmlLocalizer<ReverseProxySettingsDisplayDriver> stringLocalizer)
```

Comment:
Do you need to remove `INotifier` from here as well, now it's unused?

Going to refer you to https://github.com/OrchardCMS/OrchardCore/blob/dev/src/OrchardCore.Modules/OrchardCore.Https/Drivers/HttpsSettingsDisplayDriver.cs

for the naming conventions that are used everywhere else, for restarting tenants
ie. `shellHost`, and `shellSettings`